src/classifier_imagenet.js

```diff
@@ -187,3 +187,13 @@ tf.setBackend(process.env.RECOGNIZE_PUREJS === 'true' ? 'wasm' : 'tensorflow')
 		console.error(e)
 		process.exit(1)
 	})
+
+/**
+ *
+ * @param string
+ */
+function uppercaseWords(string) {
+	return string.split(' ').map(word => {
```

Comment:
We usually only uppercase the first word, but let's ping @jancborchardt for confirmation.

Reply:
Yep! That would be great. :) "Sentence case", not " Capital Case", as per design guidelines: https://docs.nextcloud.com/server/stable/developer_manual/design/foundations.html#wording